src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestDetailsView.tsx

```diff
@@ -92,6 +94,12 @@ const ChangeRequestDetailsView: React.FC<ChangeRequestDetailsProps> = ({
   const handleDeleteOpen = () => setDeleteModalShow(true);
   const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
   const dropdownOpen = Boolean(anchorEl);
+  const { data: project } = useSingleProject({
+    carNumber: changeRequest.wbsNum.carNumber,
+    projectNumber: changeRequest.wbsNum.projectNumber,
+    workPackageNumber: 0
+  });
+  const { name: projectName } = project || {};
```

Comment:
instead of this you can pull out isLoading on line 97 then do 

if (isLoading || !data) return <LoadingIndicator />

There should be examples of this in the codebase a lot